extensions/amp-sidebar/1.0/base-element.js

```diff
@@ -14,15 +14,21 @@
  * limitations under the License.
  */
 
+import * as Preact from '../../../src/preact';
+import {AmpSidebarToolbar, Sidebar} from './component';
 import {CSS as COMPONENT_CSS} from './component.jss';
 import {PreactBaseElement} from '../../../src/preact/base-element';
-import {Sidebar} from './component';
 import {dict} from '../../../src/core/types/object';
 import {pauseAll} from '../../../src/utils/resource-container-helper';
 import {toggle} from '../../../src/style';
 import {toggleAttribute} from '../../../src/dom';
 
 export class BaseElement extends PreactBaseElement {
+  /** @override */
+  static deferredMount(unusedElement) {
+    return false;
+  }
+
```

Comment:
Allows component render before `Sidebar` is opened.  This is needed because Toolbar Target should be rendered on page load, regardless of `Sidebar`'s `open` / `close` status.